src/Schema/ColumnDefinition.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Colopl\Spanner\Schema;
+
+use Illuminate\Database\Query\Expression;
+use Illuminate\Database\Schema\ColumnDefinition as BaseColumnDefinition;
+
+class ColumnDefinition extends BaseColumnDefinition
```

Comment:
Since `UuidColumnDefinition` is used, it does not seem necessary to add this class.

Reply:
fixed